### PR TITLE
fix flake seen in pod_shutdown_test.go

### PR DESCRIPTION
**Pull request by cfryanr**
Trying to fix a flake in a recently added test seen once here https://hush-house.pivotal.io/builds/1879066113#L6480ce4a:48:59.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by cfryanr** [comment]
This passed all CI jobs at 11 am on Tues Sept 26. I'm going to force push to run them all again to see if they pass again.

**Comment by cfryanr** [comment]
Everything passed a second time. I will try running everything for a third time.

**Comment by cfryanr** [comment]
Everything passed a third time, so maybe this PR is helping.

**Review by joshuatcasey — APPROVED** [review]

File: test/integration/pod_shutdown_test.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func testShutdownAllPodsOfApp(t *testing.T, env *testlib.TestEnv, namespace stri
 	// The leader election lease should already contain the empty string as the holder, because the old leader
 	// pod should have given up the lease during its graceful shutdown.
 	waitForLeaderElectionLeaseToHaveHolderIdentity(t, namespace, appName,
-		func(holder string) bool { return holder == "" }, 200*time.Millisecond)
+		func(holder string) bool { return holder == "" }, 1*time.Minute)
```
Comment (benjaminapetersen) [inline test/integration/pod_shutdown_test.go:123]: 5 times per second down to once a minute, quite a bit longer?

Comment (cfryanr) [reply test/integration/pod_shutdown_test.go:123]: 1 minute is how to to wait for the predicate to become true (not how often to poll during that minute).

**Pull request merged**